.github/workflows/bump-agent-versions.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
```

Comment:
Not related to this PR: Why does this workflow use `go-version: 1.21` and not `go-version-file: .go-version` as the other workflows? :thinking: